tonic-build/src/server.rs

```diff
@@ -49,7 +49,7 @@ pub fn generate<'a, T: Service<'a>>(service: &'a T, context: &T::Context) -> Tok
                 }
             }
 
-            impl<T: #server_trait> Service<http::Request<HyperBody>> for #server_service<T> {
+            impl<T: #server_trait> Service<http::Request<RequestBody>> for #server_service<T> {
```

Comment:
So instead of making this accept a BoxBody we actually want to make this generic over it. So we can do something like:

```rust
impl<T, B> Service<http::Request<B>> for #server_service<T>
where
	T: #server_trait,
	B: HttpBody + Send + Sync + 'static,
	B::Error: Into<StdError> + Send + 'static,
{ ... }
```

Reply:
Updated ! Thanks a lot for your help